packages/story-editor/src/components/header/buttons/publish.js

```diff
@@ -125,10 +125,7 @@ function PublishButton({ forceIsSaving }) {
       <ButtonWithChecklistWarning
         onClick={handlePublish}
         disabled={
-          !capabilities?.hasPublishAction ||
-          isSaving ||
-          forceIsSaving ||
-          isUploading
+          !capabilities?.publish || isSaving || forceIsSaving || isUploading
```

Comment:
somewhere here the `forceIsSaving` value got lost in a different PR? Since it's propType and prop still existed I am restoring it to disabling the publish button when true.

Reply:
`forceIsSaving` is still there right? I don't think it was removed, it was just on a different line 🤔

Reply:
This PR's been up for a while, it was missing at one point but now it is back, you're right. I'll update the order so that this diff goes away.

Reply:
oh i'm just blind. i get why this is confusing. So `forceIsSaving` _was_ removed here, adding it back in to avoid a regression - but someone else did the same so that's moot now. the capabilities did get updated here however so it's looking for `publish` not `hasPublishAction`